spring-rabbit/src/main/java/org/springframework/amqp/rabbit/config/SimpleRabbitListenerContainerFactory.java

```diff
@@ -187,6 +191,14 @@ public void setMissingQueuesFatal(Boolean missingQueuesFatal) {
 		this.missingQueuesFatal = missingQueuesFatal;
 	}
 
+	/**
+	 * @param consumerTagStrategy the consumerTagStrategy to set
+	 * @see SimpleMessageListenerContainer#setConsumerTagStrategy(ConsumerTagStrategy)
```

Comment:
`@since 1.4.5`.
I see the `@see`, but it really doesn't say which version this method has been introduced with.